src/plugins/rovpilot/public/js/rovpilot.js

```diff
@@ -379,11 +379,7 @@
       //to deal with the clocks not being in sync between the computer and the ROV.
       var self = this;
 
-      //Initial sync of state
-      self.cockpit.rov.emit('plugin.rovpilot.getState', function(state) {
```

Comment:
In MOCK mode, this fired but the callback function never fired. It fires when you "wait" a little bit and then emit this.

@BrianAdams thoughts?